packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -358,14 +396,34 @@ export const localL3NetworkRpcUrl = loadEnvironmentVariableWithFallback({
   fallback: 'http://127.0.0.1:3347'
 })
 
-export function registerLocalNetwork() {
+export async function registerLocalNetwork(isLocalCustomNativeToken?: boolean) {
```

Comment:
We fetch native token to see if it exists, this way we know it's a custom native token local chain. But we also need a way to specify it e.g. in e2e synpress setup because then fetching the token will fail